packages/destination-actions/src/destinations/amazon-amc/syncAudiencesToDSP/index.ts

```
@@ -145,6 +151,11 @@ function createPayloadToUploadRecords(payloads: Payload[], audienceSettings: Aud
   const records: AudienceRecord[] = []
   const { audienceId } = payloads[0]
   payloads.forEach((payload: Payload) => {
+    // Check if the externalUserId matches the pattern
+    if (!REGEX_EXTERNALUSERID.test(payload.externalUserId)) {
```

This destination might need multistatus. We should not silently drop events 😅

---

Yes, we can add multi-status to this destination. I've created a separate [ticket](https://segment.atlassian.net/browse/STRATCONN-5314) in the backlog for it.